assets/js/base/context/providers/cart-checkout/payment-methods/test/payment-method-data-context.js

```diff
@@ -24,7 +24,7 @@ import {
 import {
 	CheckoutExpressPayment,
 	SavedPaymentMethodOptions,
-} from '../../../../../../blocks/cart-checkout/payment-methods';
+} from '../../../../../../blocks/shared/payment-methods';
```

Comment:
Having a `shared` folder under `/blocks` seems to me like a place that will gather common logic for a diferent blocks. Should we rename it to make it cart & checkout specific? `cart-checkout-shared` maybe?

Reply:
My thinking was that anything that needs to be shared between two or more blocks could live here.

Reply:
My comment was coming from the fact that checkout related blocks are scope wise separated from the rest, so having common parts with those won’t be the case. There is also no shared folder between the non-checkout blocks at the moment, so renaming would avoid the confusion around which blocks actually use shared functionality from the folder

Reply:
Ok that makes sense, happy to go with your suggestion and rename to `cart-checkout-shared`